packages/integration-objects/srv/utils/AuditLog.js

```diff
@@ -0,0 +1,100 @@
+/* Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. */
+'use strict';
+const xsenv = require('@sap/xsenv');
+const auditLogging = require('@sap/audit-logging');
+const { logger } = require('../logging/Logger');
+
+const credentials = xsenv.getServices({ auditlog: { tag: 'auditlog' } }).auditlog;
+const xsuaa = xsenv.getServices({ xsuaa: { tag: 'xsuaa' } }).xsuaa;
+
+class AuditLog {
+  constructor() {
+    this.tenantId = xsuaa ? xsuaa.tenantid : null;
+    console.log(this);
+
+    auditLogging.v2(credentials, (err, auditLog) => {
```

Comment:
Also this seems dangerous to me. Assume it takes a while to get the audit log or it fails. Then you still export the `new AuditLog` method. If to get an audit log requires something async I would suggest to define a aysnc method  `getAuditLog` returning a promise containing the AuditLog. This can use the singleton pattern to ensure that just one audit log is returned after the promise is resolved. 

Feel free to also create an seperate issue for that.

Reply:
updated this in issues too.

Reply:
I do not see a new issue - where do you create the issue?

Reply:
https://github.com/SAP/cloud-sdk-currency-conversion/issues/38